src/BlazingPizza.ComponentsLibrary/content/localStorage.js

```diff
@@ -0,0 +1,7 @@
+﻿(function () {
+    window.blazorLocalStorage = {
+        get: key => key in localStorage ? JSON.parse(localStorage[key]) : null,
+        set: (key, value) => { localStorage[key] = JSON.stringify(value); },
+        delete: key => { delete localStorage[key]; }
+    };
+})();
```

Comment:
I'm still planning to convert this to use the new static content support. I'm happy to wait until you finish this massive PR

Reply:
Cool!